xbmc/cores/VideoPlayer/DVDSubtitles/DVDSubtitleParserMPL2.cpp

```diff
@@ -48,20 +51,24 @@ bool CDVDSubtitleParserMPL2::Open(CDVDStreamInfo &hints)
     int pos = reg.RegFind(line);
     if (pos > -1)
     {
-      const char* text = line + pos + reg.GetFindLen();
+      std::string text(line + pos + reg.GetFindLen());
```

Comment:
not needed afaics, you pass the text to AddSubtitle as a c str

Reply:
ConvertLine need a string